operator/test/test.sh

```diff
@@ -149,17 +149,27 @@ test_pipelines() {
   # change ubuntu image to ubi to avoid dockerhub registry pull limit
   PIPELINE_RUN=$(curl --fail --silent "$BASE_URL/examples/v1beta1/$manifest" | sed 's|ubuntu|registry.access.redhat.com/ubi8/ubi-minimal:latest|' | sed '/serviceAccountName/d' | KUBECONFIG="$KUBECONFIG_KCP" kubectl create -f -)
   echo "$PIPELINE_RUN"
-  
-  KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
+
+  # KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
+  local wait_period=0
+  while [[ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl get PipelineRun -o json  | jq -r '.items[0].status.conditions[0].reason')" != "Succeeded" ]]; do
+    KUBECONFIG="$KUBECONFIG_KCP" kubectl get PipelineRun -o json ## for debugging
+    if [ "$wait_period" -gt 360 ]; then
+      echo "[ERROR]Failed to execute Pipelinerun"
+      KUBECONFIG="$KUBECONFIG_KCP" get pipelinerun -o json
+      exit 1
+    fi
+    sleep 10
+    wait_period=$((wait_period + 10))
+  done
+
   echo "Print pipelines custom resources inside kcp"
   KUBECONFIG="$KUBECONFIG_KCP" kubectl get pipelineruns
   echo "Print kube resources in the physical cluster (Note: physical cluster will not know what pipelinesruns are)"
   
   KCP_NS_NAME="$(get_namespace)"
   kubectl get pods -n "$KCP_NS_NAME"
 
-  test_results
```

Comment:
Why is this test removed?

Reply:
I confirmd with @avinal that the test doesn't work for tekton results testing. It mainly failed at grpc command
>Hi Xin, I confirmed with my team, these tests (even if everything is correct) won't work for tekton results testing because there has been some core changes in the functionalities. You can safely comment them out for now.